pacman/utilities/utility_objs/resource_tracker.py

```diff
@@ -1233,7 +1233,7 @@ def allocate_resources(self, resources, chips=None,
                         x, y, processor_id))
             else:
                 raise PacmanValueError(
-                    "Processor id is not available on any of the chips"
+                    "Processor id {} is not available on any of the chips"
```

Comment:
dont you also need to define the chips. given chip constraints limit it to 1. whereas any chips is all.

Reply:
The line above deals with the typical case of core constraint as then len(chips) == 1.

Not sure if this line will every be hit but put it in just in case.

Reply:
yes i agree that your any chip list is 1. but the error message imples over all the machine

Reply:
if len(chips) == 1: ELSE this error